src/main/java/seedu/fridgefriend/exception/InvalidFoodLocationException.java

```diff
@@ -0,0 +1,16 @@
+package seedu.fridgefriend.exception;
+
+//@@author leeyp
+/**
+ * Signals that the food category does not match any existing category.
+ */
+public class InvalidFoodLocationException extends Exception {
+
+    private static final String messagePart1 = "Sorry my friend, ''";
```

Comment:
can we change this HAHA we have a lot of "Sorry my friend"s
maybe like Sorry amigo or smth